kaizen/auto/__init__.py

```
@@ -0,0 +1,243 @@
+    global _flush_registered
+
+    if _flush_registered:
+        return
+
+    def _flush():
+        try:
+            # Use global getter to avoid stale closure
+            provider = get_tracer_provider()
+            if provider:
+                provider.force_flush()
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)